Automattic/wp-calypso
# Pattern Assembler - Use the site name or the text Site Title to preview patterns

#### Proposed Changes

* Pass `site_title` with the site name to the API `/blocks-preview/pattern`

#### Testing Instructions


**Test the site name**
* Access `/setup?siteSlug={Site slug}` using an existing site
* Don't select any goal
* Select a vertical
* Click on the blank canvas CTA
* Click to choose a header and check that the site name is on the header

**Test the text Site Title**
This works because the site name by default when a new site is created is "Site Title"
* Access `/start` without the `siteSlug` params, pick a domain name and skip the plan 
* Don't select any goal or vertical
* Click on the blank canvas CTA
* Click to choose a header and check that "Site Title" is on the header


#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/69362
